src/components/standalone/firewall/conntrack/DeleteConntrackRecordModal.vue

```diff
@@ -0,0 +1,102 @@
+<!--
+  Copyright (C) 2024 Nethesis S.r.l.
+  SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import { ubusCall } from '@/lib/standalone/ubus'
+import { NeInlineNotification, getAxiosErrorMessage } from '@nethesis/vue-components'
+import { NeModal } from '@nethesis/vue-components'
+import { ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+import type { ConntrackRecord } from './ConntrackContent.vue'
+
+const { t } = useI18n()
+
+const props = defineProps<{
+  visible: boolean
+  itemToDelete?: ConntrackRecord
+}>()
+
+const emit = defineEmits(['close', 'record-deleted'])
+
+const error = ref({
+  notificationDescription: '',
+  notificationDetails: ''
+})
+const isDeleting = ref(false)
+
+async function deleteConntrackRecord() {
+  if (props.itemToDelete) {
+    error.value = {
+      notificationDescription: '',
+      notificationDetails: ''
+    }
+    isDeleting.value = true
+    try {
+      if (props.itemToDelete.id !== 'all') {
+        await ubusCall('ns.conntrack', 'drop', {
+          id: props.itemToDelete.id
+        })
+      } else {
+        await ubusCall('ns.conntrack', 'drop_all', {})
+      }
+      emit('record-deleted')
+      emit('close')
```

Comment:
This should be just one event, the `record-deleted` listener should also close the modal